session_5/readme/README.md

```diff
@@ -0,0 +1,159 @@
+SCHEMA
+```ruby
+ActiveRecord::Schema.define(version: 2021_01_30_150003) do
+
+  create_table "assignments", force: :cascade do |t|
```

Comment:
So you are not keeping track of a student's submissions to an assignment? I can't find a relevant table here.